### PR TITLE
fix(ava-react/insight-card): using spec generator from insight module

### PR DESCRIPTION
**Pull request by chenluli**
### PR includes

- import spec generator from insight module
- fix some bugs of InsighCard

- [ ] fixed #0
- [ ] add / modify test cases
- [ ] documents, demos

### CODE REVIEW

File: packages/ava-react/src/InsightCard/types.ts [hunk 1]
```diff
@@ -1,8 +1,7 @@
-import type { NtvTypes, NtvPluginType } from '@antv/ava-react';
+import type { NtvTypes, NtvPluginType } from '../NarrativeTextVis';
```
Comment (BBSQQ) [inline packages/ava-react/src/InsightCard/types.ts:1]: 这个之后要改的是吧

File: packages/ava-react/src/InsightCard/utils/specGenerator.ts [hunk 2]
```diff
@@ -1,33 +1,42 @@
...
+              customType: DISPLAY_CHARTS_PLUGIN_KEY,
+              chartSpecs,
+            },
+          ],
         },
       ],
     };
     return insightContentSpec;
   }
-  // todo 调用 insight module 中的方法生成 spec
```
[the remark this answers is not in the document]
Comment (chenluli) [reply packages/ava-react/src/InsightCard/utils/specGenerator.ts:31]: 这个完成了 从 ava 里导入了 generateInsightVisualizationSpec

**Review by BBSQQ — APPROVED** [review]
LG

File: playground/src/DevPlayground/LiteInsight/AutoInsight.tsx [hunk 3]
```diff
@@ -1,19 +1,19 @@
-import React, { useState, useEffect, createRef } from 'react';
+import React, { useState, useEffect } from 'react';
```
Comment (pddpd) [inline playground/src/DevPlayground/LiteInsight/AutoInsight.tsx:1]: Is the site demo in the next PR?

Comment (chenluli) [reply playground/src/DevPlayground/LiteInsight/AutoInsight.tsx:1]: yes, site demos and docs are not included in this PR

**Pull request merged**